src/utilities/googleAnalytics.ts

```diff
@@ -38,3 +47,53 @@ export const getNotionalActionFromHintState = (hint: SwapFormSubmitButtonHintSta
             return "";
     }
 }
+
+export const pushEvent = (eventName: string, eventValue: string | number, sessionId?: string, amm?: AugmentedAMM, agent?: string) => {
```

Comment:
```suggestion
type AnalyticEventPayload = {
 eventName: string;
 eventValue: string | number; 
 sessionId?: string;  // check if this can be required param
 amm?: AugmentedAMM;
 agent?: string; // check if we can limit the agent to certain values, similar to the eventName
}
export const pushEvent = ({ eventName, eventValue, sessionId, amm, agent }: AnalyticEventPayload) => {
```